### DISCUSSION TITLE
The tallest building in the world at 828m is ... ?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#367] [Galaxy Trivia] [answered]
**Hint:** in Dubai

**Comment by skzidan** [comment]
Burj Khalifa
The 828-metre (2,717 ft) tall Burj Khalifa in Dubai has been the tallest building since 2010. The Burj Khalifa has been classified as megatall.

**Comment by thuongtruong1009** [comment, marked as answer]
The tallest building in the world at 828 meters is the Burj Khalifa, located in Dubai, United Arab Emirates. It was completed in 2010 and has 163 floors. The Burj Khalifa held the title of the tallest building in the world since its completion, until the completion of the Jeddah Tower in Saudi Arabia, which is currently under construction and is expected to be taller than the Burj Khalifa when completed.
